balancer_conn_wrappers.go

```diff
@@ -78,6 +80,19 @@ func (ccb *ccBalancerWrapper) watcher() {
 			su := t.(*scStateUpdate)
 			ccb.balancer.UpdateSubConnState(su.sc, balancer.SubConnState{ConnectivityState: su.state, ConnectionError: su.err})
 			ccb.balancerMu.Unlock()
+		case sc := <-ccb.scToRemove.Get():
```

Comment:
Optional: re-use `scBuffer` but do a type switch instead of a type assertion.  This removes some duplication (loading, which, by the way, left you with a bug), and some allocations.

Reply:
Done